src/modules/ie-objects/elasticsearch/elasticsearch.consts.ts

```diff
@@ -64,6 +66,7 @@ export enum QueryType {
 
 export const MULTI_MATCH_QUERY_MAPPING = {
 	fuzzy: {
+		booleanQuery: booleanSearchQueryTemplateFuzzy,
```

Comment:
The other template is also used for boolean search
so we better call this one exactQuery and keep the other on "fuzzy"
```suggestion
		exactQuery: searchQueryTemplateExact,
```